src/setfit/modeling.py

```diff
@@ -132,30 +140,28 @@ def forward(
 
         return outputs
 
-    def predict_proba(self, x_test: torch.Tensor) -> torch.Tensor:
+    def predict_proba(self, x_test: Union[torch.Tensor, "ndarray"]) -> Union[torch.Tensor, "ndarray"]:
```

Comment:
I moved some functionality from `predict` to here to make them work the same way. I did this because I found it cumbersome to write the tests without it.